sdk/eventhub/event-hubs/package.json

```diff
@@ -60,7 +60,7 @@
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "generate-certs": "node ./scripts/generateCerts.js",
     "integration-test:browser": "cross-env TEST_TARGET=live DISABLE_MULTI_VERSION_TESTING=true karma start --single-run",
-    "integration-test:node": "cross-env TEST_TARGET=live DISABLE_MULTI_VERSION_TESTING=true nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 600000 --full-trace \"dist-esm/test/internal/*.spec.js\" \"dist-esm/test/public/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\" \"dist-esm/test/internal/**/*.spec.js\"",
+    "integration-test:node": "cross-env TEST_TARGET=live DISABLE_MULTI_VERSION_TESTING=true dev-tool run test:node-js-input -- --timeout 600000 \"dist-esm/test/internal/*.spec.js\" \"dist-esm/test/public/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\" \"dist-esm/test/internal/**/*.spec.js\"",
```

Comment:
Using the dev-tool command essentially runs the test-proxy tool for no reason since there are no recorded tests for the AMQP packages.

I don't really mind the change, just pointing out the unnecessary additional step here.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/24731/commits/8f152da300ad3d8bf963184baf35d517ff2ebf8a.